docs/07-BDD.md

```diff
@@ -1,90 +1,113 @@
+what has not, what has been tested and what has not.
 
 ### Ubiquitous Language
 
-The ubiquitous language is always referred as *common* language. That is it's main benefit. It is not a couple of our business specification's words, and not a couple of developer's technical terms. It is a common words and terms that can be understood by people for whom we are building the software and should be understood by developers. Establishing correct communication between this two groups people is vital for building successful project that will fit the domain and fulfill all business needs.
+The ubiquitous language is always referred as *common* language. That is its main benefit.
+It is not some of our business specification's words, and some of the developer's technical terms.
+It is a common words and terms that can be understood by people for whom we are building the software
+and should be understood by developers. Establishing correct communication between these two groups people is vital
+for building a successful project that will fit the domain and fulfil all business needs.
 
-Each feature of a product should be born from a talk between
+Each feature of a product should be born from a talk between:
 
 * business (analysts, product owner)
 * developers
 * QAs
 
-which are known in BDD as "three amigos".
+who are known in BDD as the "three amigos".
 
-Such talks should produce written stories. There should be an actor that doing some things, the feature that should be fulfilled within the story and the result achieved. 
+Such talks should produce written stories. There should be an actor that want to do some things,
+the feature that should be fulfilled within the story and the result achieved.
 
-We can try to write such simple story:
+As an example, consider the following story:
 
 ```
 As a customer I want to buy several products
-I put first product with 600 $ price to my cart
-And then another one with 1000 $ price
-When I go to checkout process
-I should see that total number of products I want to buy is 2
-And my order amount is 1600 $
+I add a $600 product to my cart
+And then a $1000 one
+When I go to checkout
+I should see that the total number of products is 2
+And my order amount is $1600
 ```
 
-As we can see this simple story highlights core concepts that are called *contracts*. We should fulfill those contracts to model software correctly. But how we can verify that those contracts are being satisfied? [Cucumber](http://cucumber.io) introduced a special language for such stories called **Gherkin**. Same story transformed to Gherkin will look like this:
+As we can see this simple story highlights core concepts that are called *contracts*.
+We should fulfil those contracts to model the software correctly.
+But how we can verify that those contracts are being satisfied?
+[Cucumber](http://cucumber.io) introduced a special language for such stories called **Gherkin**.
```

Comment:
@damko Is there any way you could make those changes directly instead of commenting? Otherwise I have a huge amount of effort duplicating your improvements.

I don't expect that each one of my changes is perfect, merely better than before. Other people are welcome to make further changes. I have included some things I wasn't 100% sure about because anyone with more knowledge than me can correct it later.